internal/server.go

```diff
@@ -20,16 +19,16 @@ func logJSONError(w http.ResponseWriter, err error, code int) {
 }
 
 func parseQueryParameters(urlQuery url.Values) (queryParameters, error) {
-	if !urlQuery.Has("interval") {
-		return queryParameters{}, errors.New("Interval parameter is missing or is not valid.")
-	}
-	if !urlQuery.Has("step") {
-		return queryParameters{}, errors.New("Step parameter is missing or is not valid.")
-	}
+	// if !urlQuery.Has("interval") {
+	// 	return queryParameters{}, errors.New("Interval parameter is missing or is not valid.")
+	// }
```

Comment:
No need to comment them out, we have version control system if we want to recover from any problem anyway